src/app/content/components/BookBanner.tsx

```diff
@@ -191,6 +197,7 @@ export class BookBanner extends Component<PropTypes, {scrollTransition: boolean}
       this.setState({
         scrollTransition: miniRect.top === 0 &&
           this.bigBanner.current.offsetTop + this.bigBanner.current.clientHeight > window.scrollY,
+        tabbableBanner: miniRect.top === 0 ? 'mini' : 'big',
```

Comment:
did you have to add a new property here? can't you use the existing `scrollTransition` to switch the tabbing around?

Reply:
`scrollTransition` seems to be true only when the banners are transitioning, i think? It's set back to false after scrolling down further